input/definitions/MeasureReport/StructureDefinition-criteriaReference.json

```diff
@@ -0,0 +1,88 @@
+{
+  "resourceType": "StructureDefinition",
+  "id": "criteriaReference",
+  "url": "http://hl7.org/fhir/StructureDefinition/criteriaReference",
+  "version": "1.0.0",
+  "name": "CriteriaReferenceExtension",
+  "title": "Criteria Reference Extension",
+  "status": "active",
+  "experimental": false,
+  "date": "2023-12-15",
+  "publisher": "HL7 International / Clinical Decision Support",
```

Comment:
This should actually be CQI, they own MeasureReport, so are the stewards of extensions that reference it:
"HL7 International / Clinical Quality Information"

Reply:
```suggestion
  "publisher": "HL7 International / Clinical Quality Information",
```